### PR TITLE
Merging Dan Coughlin's and Christina Lin's updates to the Fuse getting started page

### PR DESCRIPTION
**Pull request by fbolton**

### CODE REVIEW

**Comment by Dantheman720** [comment]
retest this please.

File: products/fuse/get-started.adoc [hunk 1]
```diff
@@ -1,190 +1,134 @@
...
+
+* Java SE Development Kit (JDK) version 1.8.x (Java 8) - we recommend Oracle JDK or OpenJDK
+* https://maven.apache.org/download.cgi[Apache Maven]
+
+[[fuse-karaf-install-runtime]]
+=== Install and configure the JBoss Fuse Karaf runtime
+video::183851396[vimeo]
+
+. Download the {fuse-on-karaf-download-url}[JBoss Fuse Karaf] runtime package.
+. Unpack the downloaded `.zip` file to a convenient location on your file system, `_FuseInstall_`.
```
Comment (Dantheman720) [inline products/fuse/get-started.adoc]: Do we have a link for this?

File: products/fuse/get-started.adoc [hunk 2]
```diff
@@ -219,4 +163,93 @@ Where do you go from here?
...
 |===
 
+## EAP Runtime
+
+=== Prerequisites
+Before you start, make sure you have installed the following software:
+
+* Java SE Development Kit (JDK) version 1.8.x (Java 8) - we recommend Oracle JDK or OpenJDK
+* https://maven.apache.org/download.cgi[Apache Maven]
+* JBoss Fuse on EAP runtime - see {installation-on-eap-link}/[Installation on JBoss EAP]
```
Comment (Dantheman720) [inline products/fuse/get-started.adoc]: Do we have a link for this?

File: products/fuse/get-started.adoc [hunk 3]
```diff
@@ -219,4 +163,93 @@ Where do you go from here?
...
+
+=== Set up your development environment
+
+The following steps will install a local version of JBoss Developer Studio 9.1.0 (the Eclipse Mars edition) along with the Fuse tooling.
+
+. Download the {jbdsis-standalone-installer-download-url}[Red Hat JBoss Developer Studio Integration Stack 9.0.2.GA Stand-Alone Installer].
```
Comment (Dantheman720) [inline products/fuse/get-started.adoc]: Do we have a link for this?

File: products/fuse/get-started.adoc [hunk 4]
```diff
@@ -1,190 +1,134 @@
...
+----
+USERNAME=PASSWORD,admin,manager,viewer,Operator, Maintainer, Deployer, Auditor, Administrator, SuperUser
+----
++
+Save the `users.properties` file.
+
+=== Set up your development environment
+video::183852576[vimeo]
+
+. Download the {jbdsis-standalone-installer-download-url}[Red Hat JBoss Developer Studio Integration Stack 9.0.2.GA Stand-Alone Installer].
```
Comment (Dantheman720) [inline products/fuse/get-started.adoc]: Do we have a link for this?

Comment (fbolton) [inline products/fuse/get-started.adoc]: Just to clarify, this link (and the preceding links) are defined by the following variables, which are set at the top of the get-started.adoc file:

`:jbdsis-standalone-installer-download-url: https://developers.redhat.com/download-manager/file/devstudio-integration-stack-9.0.2.GA-standalone-installer.jar
:fuse-on-karaf-download-url: https://developers.redhat.com/download-manager/file/jboss-fuse-karaf-6.3.0.redhat-187.zip
:installation-on-eap-link: https://access.redhat.com/documentation/en/red-hat-jboss-fuse/6.3/paged/installation-on-jboss-eap`

**Comment by LightGuard** [comment]
@fbolton @Dantheman720 is this good to merge now?

**Comment by fbolton** [comment]
@LightGuard Let me just check if those download links are good

**Comment by fbolton** [comment]
@LightGuard @Dantheman720 The link to the developer studio standalone installer is good, but the link to download the JBoss Fuse Karaf runtime is 404ing:

https://developers.redhat.com/download-manager/file/jboss-fuse-karaf-6.3.0.redhat-187.zip

**Comment by fbolton** [comment]
@LightGuard @Dantheman720 The download links now appear to be working, so I believe we can go ahead and merge the new getting started content.

**Comment by Dantheman720** [comment]
@adelasofia @LightGuard @paulrobinson - I'm not sure if I'm able to merge this since I opened the initial PR. Can you review/merge this?

http://developers-pr.stage.redhat.com/pr/1496/drupal/369/products/fuse/get-started/

**Comment by fbolton** [comment]
@Dantheman720 Actually, I was the one who created this PR (I created a new one, instead of continuing your one). So I think you might be able to merge this.

**Pull request merged**